src/ResultView.tsx

```diff
@@ -11,6 +12,34 @@ const ResultView = ({ result }: { result: any }) => {
     school,
   } = result;
 
+  var str = String(role.snippet);
+  var open = 0;
+  var close = 0;
+  for (var i = 0; i < str.length; i++) {
+    if (str.charAt(i) === "(") {
+      open += 1;
+    } else if (str.charAt(i) === ")") {
+      close += 1;
+    }
+  }
+  if (open - close === 1) {
+    role.snippet = role.snippet + ")";
+  }
```

Comment:
Could this be abstracted into a function, along with a comment on why this hack is being done? Also, it's acceptable to use `==` in place of `===` here.